docs/sources/k6/next/using-k6-browser/how-to-write-browser-tests/asynchronous-operations.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Asynchronous operations
+description: 'Learn how the k6 browser module uses asynchronous operations.'
+weight: 100
+---
+
+# Asynchronous operations
+
+Since many browser operations happen asynchronously, and to follow the Playwright API more closely, we are working on migrating most of the browser module methods to be asynchronous as well.
```

Comment:
This has been completed. Most of the browser module APIs are now async and require working with `await` keyword.

The reason we made them async was:
1. To be more aligned with PW;
2. In JS to avoid blocking the single event loop, any long running processes or IO operations should be made async so that they can be handled in the background. Most of the browser module APIs are IO operations since they communicate with chromium over a websocket.

There were other reasons, but these two are the most important.